python/tskit/trees.py

```diff
@@ -1004,8 +1004,17 @@ def draw(
         :param map node_labels: If specified, show custom labels for the nodes
             that are present in the map. Any nodes not specified in the map will
             not have a node label.
-        :param map node_colours: If specified, show custom colours for nodes. (Only
-            supported in the SVG format.)
+        :param map node_colours: If specified, show custom colours for the nodes
+            given in the map. (Only supported in the SVG format.)
+        :param map edge_colours: If specified, show custom colours for the edge
```

Comment:
I think we should follow the same semantics as ``node_labels``: if the node isn't present, don't draw the edge, if it maps to None, use the default colour. I think the description would be clearer as "If specified, show custom colours for the edge joining each node in the map to its parent".

Reply:
Re semantics, see discussion at 
https://github.com/tskit-dev/tskit/issues/149#issuecomment-474246468 . I will change the description accordingly